packages/manager/modules/netapp/package.json

```diff
@@ -28,6 +28,7 @@
     "@ovh-ux/ng-at-internet-ui-router-plugin": "^3.1.2",
     "@ovh-ux/ng-ovh-utils": "^14.0.14",
     "@ovh-ux/ng-translate-async-loader": "^2.1.0",
+    "@ovh-ux/ng-ui-router-breadcrumb": "^1.1.4",
```

Comment:
Maybe add to generator ?

Reply:
Good point! I will handle it separately (new Pull Request).

Reply:
Please refer to #5430